app/prisma/schema.prisma

```diff
@@ -167,6 +167,7 @@ model PruningRuleMatch {
 
 model Project {
     id   String @id @default(uuid()) @db.Uuid
+    slug String @unique() @default(dbgenerated("nanoid()")) @db.VarChar(10)
```

Comment:
I should have commented on this earlier, sorry for not catching. In Postgres there's no efficiency advantage to a `VarChar` over the standard `text` type, the only thing it does is add a constraint. I generally prefer to just stick to `text` for everything given that as it's more flexible and better supported. Not worth changing at this point though imo.